packages/template/vite-typescript/tmpl/main.ts

```diff
@@ -0,0 +1,58 @@
+import { app, BrowserWindow } from 'electron';
+import path from 'path';
+// This allows TypeScript to pick up the magic constants that's auto-generated by Forge's Vite
+// plugin that tells the Electron app where to look for the Vite-bundled app code (depending on
+// whether you're running in development or production).
+declare const MAIN_WINDOW_VITE_DEV_SERVER_URL: string;
+declare const MAIN_WINDOW_VITE_NAME: string;
```

Comment:
Is it possible to write these two types to `d.ts` here? 🤔

Reply:
I just created `types.d.ts` file, because the `MAIN_WINDOW_VITE_DEV_SERVER_URL` can useable in any files 😅